source/documentation/geosparql/geosparql-fuseki.md

```diff
@@ -3,50 +3,86 @@ title: GeoSPARQL Fuseki
 
-- 1) Conversion of Geo Predicates to the GeoSPARQL data structure (encoding the Lat/Lon as a Point geometry).
-- 2) Spatial extension which provides property and filter functions accepting Lat/Lon arguments.
+GeoSPARQL refers to the Geographic Markup Language (GML) as one format for
+`GeometryLiterals`. This does not mean that GML is part of the GeoSPARQL
+standard. Instead a subset of geometry encodings from the GML standards are
+permitted (specifically the `GML 2.0 Simple Features Profile (10-100r3)` is
+supported by GeoSPARQL Jena). The expected encoding of data is in RDF triples
+and can be loaded from any RDF file format supported by Apache Jena. Conversion
+of GML to RDF is out of scope of the GeoSPARQL standard and Apache Jena.
 
-The Spatial extension functions (documented in the [GeoSPARQL Jena module](index)) support triples in either GeoSPARQL data structure or Geo Predicates. Therefore, converting a dataset to GeoSPARQL will not lose functionality. By converting to the GeoSPARQL data structure, datasets can include a broader range of geospatial data.
+### Geo Predicates Lat/Lon
 
-## Getting Started
-GeoSPARQL Fuseki can be accessed as an embedded server using Maven etc. from Maven Central or run from the command line.
-SPARQL queries directly on Jena Datasets and Models can be done using
-the [GeoSPARQL Jena module](index).
+Historically, geopsatial data has frequently been encoded as Latitude/Longitude
+coordinates in the WGS84 coordinate reference system. The GeoSPARQL standard
+specifically chooses not to adopt this approach and instead uses the more
+versatile `GeomtryLiteral`, which permits multiple encoding formats that support
+multiple coordinate reference systems and geometry shapes. Therefore, Lat/Lon
+Geo Predicates are not part of the GeoSPARQL standard. However, GeoSPARQL Jena
+provides two methods to support users with geo predicates in their geospatial
+data.
 
-    <dependency>
-      <groupId>org.apache.jena</groupId>
-      <artifactId>fuseki-geosparql</artifactId>
-      <version>...</version>
-    </dependency>
+1. Conversion of Geo Predicates to the GeoSPARQL data structure (encoding the Lat/Lon as a Point geometry).
+2. Spatial extension which provides property and filter functions accepting Lat/Lon arguments.
 
-or download the binary from the 
-[Maven central repository org/apache/jena/jena-fuseki-geosparql](http://central.maven.org/maven2/org/apache/jena/jena-fuseki-geosparql/)
+The Spatial extension functions (documented in the [GeoSPARQL Jena module](index)) support triples in either GeoSPARQL data structure or Geo Predicates. Therefore, converting a dataset to GeoSPARQL will not lose functionality. By converting to the GeoSPARQL data structure, datasets can include a broader range of geospatial data.
```

Comment:
>documented in the [GeoSPARQL Jena module](index))

Does it need to be index.html here?

Reply:
The "html" ? Yes, sort of. It makes hugo work locally. While the site deployment adds ".html", the Hugo built in server is more limited.

Reply:
Ah, that makes sense. Asked because I saw another file in this PR with `index.html`, but I didn't try locally (TodayILearned, and useful tip since I'm migrating my blog from Jekyll to Hugo over these holidays). Thanks!